src/pages/index.js

```diff
@@ -227,107 +292,58 @@ addNewCardButton.addEventListener("click", () => {
-  return card.getView();
-}
 
-function renderCard(cardData) {
-  console.log(cardData);
-  const cardElement = getCardElement(cardData);
-  section.addItem(cardElement);
-}
+const profileEditButton = document.querySelector("#profile-edit-button");
+profileEditButton.addEventListener("click", () => {
+  const userData = userInfo.getUserInfo();
+  profileTitleInput.value = userData.name;
+  profileDescriptionInput.value = userData.description;
+  popupWithEditProfileForm.open();
+});
 
-function handleDeleteCard(cardInstance) {
-  cardDeleteModal.open();
-  cardDeleteModal.setSubmitAction(() => {
-    cardDeleteModal.setLoading(true, "Deleting...");
-    api
-      .deleteCard(cardInstance.id)
-      .then(() => {
-        cardInstance.handleDeleteButton();
-        cardDeleteModal.close();
-      })
-      .catch((err) => {
-        console.error(err);
-      })
-      .finally(() => {
-        cardDeleteModal.setLoading(false, "Yes");
-      });
-  });
-}
+const avatarForm = document.querySelector("#avatar-edit-form");
 
-function handleLike(cardInstance) {
-  if (cardInstance.isLiked) {
-    api
-      .dislikeCard(cardInstance.id)
-      .then((data) => {
-        cardInstance.setIsLiked(false);
-      })
-      .catch((err) => {
-        console.error(err);
-      });
-  } else {
-    api
-      .likeCard(cardInstance.id)
-      .then((data) => {
-        cardInstance.setIsLiked(true);
-      })
-      .catch((err) => {
-        console.error(err);
-      });
-  }
-}
-/*Event Handlers*/
+const avatarFormValidator = new FormValidator(config, avatarForm);
+avatarFormValidator.enableValidation();
 
-function handleProfileEditSubmit({ title, description }) {
-  console.log(title, description);
-  userInfo.setUserInfo({
-    name: title,
-    description: description,
-  });
-  console.log(userInfo.getUserInfo());
-  popupWithEditProfileForm.close();
-}
+const addCardButton = document.querySelector("#modal__close");
+addCardButton.addEventListener("click", () => {
+  addCardModal.open();
+});
 
-function handleAddCardFormSubmit(inputValue) {
-  const cardData = {
-    name: inputValue.title,
-    link: inputValue.url,
-  };
-  renderCard(cardData);
-  addCardFormElement.reset();
-  addFormValidator.disableButton();
-  popupWithAddCardForm.close();
-}
+const avatarButton = document.querySelector(".profile__avatar");
+avatarButton.addEventListener("click", () => {
+  profileAvatarModal.open();
+});
 
-//Validation
+// Select the form elements
+const cardAddForm = document.querySelector("#add-card-form");
 
-const editFormValidator = new FormValidator(config, profileEditForm);
+// Form validation
+const addCardFormValidator = new FormValidator(config, cardAddForm);
+addCardFormValidator.enableValidation();
+
+const profileEditFormValidator = new FormValidator(config, profileEditForm);
+profileEditFormValidator.enableValidation();
+
+
+
+deleteModalCloseButton.addEventListener("click", () => {

```

Comment:
* One more remark hasn't been corrected https://github.com/hadz44/se_project_aroundtheus/pull/4#discussion_r1971148087  Please, delete the close icon handler from `index.js` because everything is done by `Popup`